module/input/ImageDecompressor/src/decompressor/turbojpeg/Decompressor.cpp

```diff
@@ -54,10 +53,10 @@ namespace module::input::decompressor::turbojpeg {
         tjDecompressHeader2(decompressor, const_cast<uint8_t*>(data.data()), data.size(), &width, &height, &subsamp);
 
         // Work out what we decode as
-        TJPF code = mosaic || subsamp == TJSAMP::TJSAMP_GRAY ? TJPF::TJPF_GRAY : TJPF::TJPF_RGBA;
+        TJPF code = mosaic || subsamp == TJSAMP::TJSAMP_GRAY ? TJPF::TJPF_GRAY : TJPF::TJPF_RGB;
```

Comment:
This will break old recordings right?

Reply:
Nah just makes future decompressions decompress jpegs (that don't have an alpha channel) as RGB, and not as RGBA